### DISCUSSION TITLE
Is there a way to select multiple measures in a list box. Currently, using "ScriptHelper.SelectMeasure" but it only allows the user to select one.

### DISCUSSION THREAD

**Discussion by dpesante** [TabularEditor/TabularEditor3#885] [Other Topics]
I would like to implement a way for the user to be prompted to select multiple measures (or columns) using a multi-select listbox (or preferably a checkbox list). Currenty I am using "ScriptHelper.SelectMeasure" but it only allows me to select a single value at a time.

**Comment by DBojsen** [comment]
Hi @dpesante 

You can do it like this:
```
var selectedMeasures = ScriptHelper.SelectObjects(Model.AllMeasures);
foreach (var measure in selectedMeasures) {
	Info(measure.Name);
}
```

BR David - TE3 support

**Reply by dpesante** [reply]
@DBojsen thanks for the quick response. It works great. 

Couple of things that would be ideal:
1) Is there a way to change this to be a checkbox list?
2) Is there a way to "pre-select" some items (e.g. where the name of the measure = "Date")
